### DISCUSSION TITLE
Audio to be implemented for next release ?

### DISCUSSION THREAD

**Discussion by docop** [neutrinolabs/xrdp#2325] [Q&A] [answered]
Hi  So has the next version of ubuntu and other distro are all running and finally shifting to pipewire instead of the pulseaudio stuff : Will it be possible to have xrdp to support audio directly ? Without to compiling extra and fight it as it stop working.. as soon we do an update of the system.

Thanks in advance

**Comment by matt335672** [comment, marked as answer]
@docop - have a look at #2023 for where we are in this area. We're going to need some input from knowledgeable people to move this further forwards.

**Comment by docop** [comment]
Great to see it might come. While i'm not into the project, perhaps having a look at DLNA. that creating a <virtual soundcard< and then grab the streamoutput and encode to rdp protocol perhaps..   if that can give some clues..
Thanks
